scripts/generate_vif/constants.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Many scripts include a verbose description of what they are used for, I think it may make sense to have a couple paragraphs for this as well, just to give an idea of what this is for to a casual user looking around in the scripts directory.

About that, how would you feel about calling the directory "generate_usb_vif"? Adding the "usb" qualifier  may help give an idea of what this is for at a glance ("something something USB").

Reply:
Renamed directory to generate_usb_vif and added some info about each module